heketi-centos-ci-tests.sh

```diff
@@ -22,8 +33,6 @@ yum -y install \
 	qemu-kvm \
 	qemu-kvm-tools \
 	qemu-img \
-	sclo-vagrant1-vagrant \
-	sclo-vagrant1-vagrant-libvirt \
```

Comment:
Perhaps converge this 'yum install' and the one above instead of invoking 'yum install' twice?